### ISSUE TITLE
task_compiler test is not stable

### ISSUE THREAD

**Issue by k-okada** [jsk-ros-pkg/jsk_planning#55]
Even on indigo: https://travis-ci.org/jsk-ros-pkg/jsk_planning/jobs/202469672

**Comment by k-okada** [comment]
humm, give up -> https://travis-ci.org/jsk-ros-pkg/jsk_planning/builds/202469670

do you use piped open or something like that within task_compiler? In early days of roseus, we had trouble when we call other program using pipe-fork and that process send signal, roseus catch that and exit.

```

Full test results for 'task_compiler/test_results/task_compiler/rosunit-test_task_compiler.xml'
-------------------------------------------------
<testsuite tests="3" failures="1" disabled="0" errors="0" time="13" name="AllTests">
  <testcase name="all-test" status="run" time="0" classname="all-test">
   <failure message="signal-handler 2" type="AssertionError">
Test:""
Trace:0
Message:"signal-handler 2"
   </failure>
  \</testcase>
  <testcase name="test-task-compiler" status="run" time="13" classname="test-task-compiler">
  </testcase>
  <testcase name="test-task-compiler-hook" status="run" time="0" classname="test-task-compiler-hook">
  </testcase>
</testsuite>
``

**Comment by k-okada** [comment]
Closed via #49
